frontend/src/components/UserQuizzes/QuizCard.tsx

```diff
@@ -22,7 +22,8 @@ export const _QuizCard = (props: Props) => {
       <div className="flex justify-end">
         <CardActions>
           <Button
-            variant="contained"
+            variant={props.quiz.is_taken ? 'outlined' : 'contained'}
+            disabled={props.quiz.is_taken ? true : false}
```

Comment:
`props.quiz.is_taken` is a boolean, no need to put ternary
```suggestion
            disabled={props.quiz.is_taken}
```